src/app/trips/shared/trip.service.ts

```diff
@@ -56,6 +56,18 @@ export class TripService {
       );
   }
 
+  // delete an existing trip
+  deleteTrip(id: number) {
```

Comment:
make sure you use typescript and state what you will be returning. It should be Observable<SomeType>. Be sure to not use `any`.